daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md

```diff
@@ -24,14 +24,17 @@ spec:
     value: "localhost:6650"
   - name: enableTLS
     value: "false"
+  - name: token
+    value: "eyJrZXlJZCI6InB1bHNhci1wajU0cXd3ZHB6NGIiLCJhbGciOiJIUzI1NiJ9.eyJzd"
 ```
 
 ## Spec metadata fields
 
 | Field              | Required | Details | Example |
 |--------------------|:--------:|---------|---------|
-| host               | Y  | Address of the Pulsar broker. Default is `"localhost:6650"` | `"localhost:6650"`|
+| host               | Y  | Address of the Pulsar broker. Default is `"localhost:6650"` | `"localhost:6650"` OR `"http://pulsar-pj54qwwdpz4b-pulsar.ap-sg.public.pulsar.com:8080"`|
 | enableTLS          | N  | Enable TLS.  Default: `"false"` | `"true"`, `"false"`|
+| token          | N  | Enable Authentication.  |-| 
```

Comment:
Could we add a reference to the Pulsar docs here on exactly how to create this token. Is this link correct https://pulsar.apache.org/docs/en/security-jwt/#generate-tokens ? Is so, can you add a sentence in the example referencing this link?